src/App.tsx

```diff
@@ -145,6 +148,15 @@ function app(): JSX.Element {
             <Route path="/requests/:orgId" element={<Requests />} />
             <Route path="/orgdash/:orgId" element={<OrganizationDashboard />} />
             <Route path="/orgpeople/:orgId" element={<OrganizationPeople />} />
+            <Route path="/orgtags/:orgId" element={<OrganizationTags />} />
+            {/* <Route
+              path="orgtags/:orgId/orgtagdetails/:tagId"
+              element={<OrganizationTagDetails />}
+            />
+            <Route
+              path="/orgtags/:orgId/orgtagChildTags/:tagId"
+              element={<OrganizationTagChildTags />}
+            /> */}
```

Comment:
Why is this commented out? Is this something that's planned for future?

Reply:
the commented out code will be used in the upcoming PRs.